docs/_static/wharf-ci-schema.json

```diff
@@ -0,0 +1,501 @@
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": [
+                        "chart",
+                        "name",
+                        "namespace"
+                    ],
+                    "properties": {
+                        "chart": {
+                            "type": "string",
+                            "examples": [
+                                "web-app"
+                            ]
+                        },
+                        "name": {
+                            "type": "string",
+                            "examples": [
+                                "test-deploy"
+                            ]
+                        },
+                        "namespace": {
+                            "type": "string",
+                            "examples": [
+                                "stage"
+                            ]
+                        },
+                        "repo": {
+                            "type": "string",
+                            "default": "https://harbor.dgc.local/chartrepo"
+                        },
+                        "set": {
+                            "type": "array",
+                            "items": {
+                                "type": "object",
+                                "minProperties": 1,
+                                "maxProperties": 1
+                            },
+                            "examples": [
+                                [
+                                    {
+                                        "image.tag": "latest"
+                                    }
+                                ]
+                            ]
+                        },
+                        "files": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            },
+                            "examples": [
+                                [
+                                    "deploy/values.yaml"
+                                ]
+                            ]
+                        },
+                        "chartVersion": {
+                            "type": "string",
+                            "examples": [
+                                "0.1.22"
+                            ]
+                        },
+                        "helmVersion": {
+                            "type": "string",
+                            "examples": [
+                                "v2.14.1",
+                                "v3.0.2",
+                                "v3.2.0",
+                                "latest"
+                            ]
+                        },
+                        "cluster": {
+                            "type": "string",
+                            "examples": [
+                                "kubectl-config",
```

Comment:
Maybe there should be different examples.

Reply:
Changed to `stage-cluster-config` and `prod-cluster-config` to mimic #1